charts/spire/charts/tornjak-frontend/templates/NOTES.txt

```diff
@@ -8,6 +8,8 @@ Namespace: {{ include "tornjak-frontend.namespace" . }}
 Tornjak UI (Frontend)
   image: {{ template "spire-lib.image" (dict "appVersion" $.Chart.AppVersion "image" .Values.image "global" .Values.global) }}
   pull policy: {{ .Values.image.pullPolicy }}
+  SPIRE health check enabled:  "{{ .Values.spireHealthCheck.enabled }}"
+  Health check frequency [sec]: "{{ .Values.spireHealthCheck.frequencySeconds }}"
```

Comment:
Not sure if you had a specific reason to add the quotes, but this might render more nicely without quotes.

```suggestion
  SPIRE health check enabled:  {{ .Values.spireHealthCheck.enabled }}
  Health check frequency [sec]: {{ .Values.spireHealthCheck.frequencySeconds }}
```